lib/global-styles.php

```diff
@@ -111,9 +111,7 @@ function_exists( 'gutenberg_is_edit_site_page' ) &&
 	}
 	$consolidated = WP_Theme_JSON_Resolver_Gutenberg::get_merged_data( $settings, $origin );
 
-	if ( 'mobile' === $context ) {
-		$settings['__experimentalStyles'] = $consolidated->get_raw_data()['styles'];
-	}
+	$settings['__experimentalStyles'] = $consolidated->get_raw_data()['styles'];
```

Comment:
Asking for my own benefit here :)

I was wondering, is there a use case for where we'd want to access `__experimentalStyles` in the site editor, or in other words, do any templates in the site editor need to know about user styles?

Or would it be okay to limit adding this object to editor/widget contexts?

Reply:
Can the `__experimentalStyles` data be limited to the `mobile` and `post-editor` contexts? Perhaps also in the `widgets` editor if it uses the existing block-editor hooks (such as line-height), but I'm not sure how that works. The `site-editor` already has access to this data, so it wouldn't need it.

Reply:
> I was wondering, is there a use case for where we'd want to access __experimentalStyles in the site editor

Yes on the site editor blocks still need to access the __experimentalStyles flag, as the hooks like line-height also run there. But as @oandregal said on the site editor we already have this information so we don't need to pass it again, I'm going to update the code. The setting will be available anyway on the block editor it will just be dynamically computed.